### PR TITLE
Reviewed header files for new APIs

### PR DESCRIPTION
**Pull request by avladimi**
Reviewed Doxygwn comments in header files for new OpenVINO APIs

### CODE REVIEW

File: src/inference/include/openvino/runtime/compiled_model.hpp [hunk 1]
```diff
@@ -52,158 +52,158 @@ class OPENVINO_RUNTIME_API CompiledModel {
...
 
     /**
-     * @brief Gets output of a compiled model identified by an @p index
-     * An output contain information about output tensor such as tensor shape, names and element type
-     * @param i An output index
-     * @return A compiled model output
-     * @note The method throws ov::Exception if output with specified index @p index is not found
+     * @brief Gets output of a compiled model identified by @p index.
+     * The output contains information about output tensor such as tensor shape, names, and element type.
+     * @param i Output index.
```
Comment (ilya-lavrenov) [inline src/inference/include/openvino/runtime/compiled_model.hpp:130]: index of output?

File: src/inference/include/openvino/runtime/compiled_model.hpp [hunk 2]
```diff
@@ -52,158 +52,158 @@ class OPENVINO_RUNTIME_API CompiledModel {
...
 
     /**
-     * @brief Gets input of a compiled model identified by an @p i
-     * An input contains information about input tensor such as tensor shape, names and element type
-     * @param i An input index
-     * @return A compiled model input
-     * @note The method throws ov::Exception if input with specified index @p i is not found
+     * @brief Gets input of a compiled model identified by @p i.
+     * The input contains information about input tensor such as tensor shape, names, and element type.
+     * @param i Input index.
```
Comment (ilya-lavrenov) [inline src/inference/include/openvino/runtime/compiled_model.hpp:95]: index of input?

File: src/inference/include/openvino/runtime/compiled_model.hpp [hunk 3]
```diff
@@ -54,125 +54,125 @@ class OPENVINO_RUNTIME_API CompiledModel {
...
     /**
-     * @brief Gets a single input of a compiled model
-     * An input is represented as an output of ov::op::v0::Parameter operation.
-     * An input contain information about input tensor such as tensor shape, names and element type
-     * @return A compiled model input
-     * @note If a model has more than one input, this method throws an ov::Exception
+     * @brief Gets a single input of a compiled model.
+     * The input is represented as an output of the ov::op::v0::Parameter operation.
+     * The input contains information about input tensor such as tensor shape, names, and element type.
+     * @return A compiled model input.
```
Comment (ilyachur) [inline src/inference/include/openvino/runtime/compiled_model.hpp:89]: Below you changed the same line to `Compiled model input.`

File: src/inference/include/openvino/runtime/compiled_model.hpp [hunk 4]
```diff
@@ -54,125 +54,125 @@ class OPENVINO_RUNTIME_API CompiledModel {
...
     /**
-     * @brief Sets properties for current compiled model
+     * @brief Sets properties for the current compiled model.
      *
-     * @tparam Properties Should be the pack of `std::pair<std::string, ov::Any>` types
-     * @param properties Optional pack of pairs: (property name, property value)
-     * @return nothing
+     * @tparam Properties Should be the pack of `std::pair<std::string, ov::Any>` types.
+     * @param properties Optional pack of pairs: (property name, property value).
+     * @return Nothing.
```
Comment (ilyachur) [inline src/inference/include/openvino/runtime/compiled_model.hpp:175]: Should we just remove @return?

Comment (ilya-lavrenov) [reply src/inference/include/openvino/runtime/compiled_model.hpp:175]: agree

Comment (avladimi) [reply src/inference/include/openvino/runtime/compiled_model.hpp:175]: @ilya-lavrenov  @ilyachur Should I remove all similar instances across all headers files?

**Review by andrew-zaytsev — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**